### PR TITLE
feat: add discover feed form from rsshub

### PR DESCRIPTION
**Pull request by Innei**
# 🎉 Feature

More user-friendly RSSHub parameter input, import discovered feed from RSSHub

https://github.com/RSSNext/follow/assets/41265413/b4ec1c9f-198b-46f2-9c05-32628087c73a

### CODE REVIEW

File: src/renderer/src/modules/discover/types.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export type RSSHubRouteType = Record<string, RSSHubRouteDeclaration>
+export interface RSSHubRouteDeclaration {
+  routes: Routes
+  name: string
+  url: string
+}
+type Routes = Record<string, RSSHubRoute>
+export type RSSHubRoute = {
+  path: string
```
Comment (TonyRL) [inline src/renderer/src/modules/discover/types.ts:9]: path can be `string[]` as well. 
https://github.com/RSSNext/follow/blob/9301a186d60b76b3dfe57bd7eb127707b9d72da9/src/renderer/src/modules/discover/example-data.json#L896-L898

Comment (Innei) [reply src/renderer/src/modules/discover/types.ts:9]: Yes, but we've done the processing at the server to prioritize the first item, so it's always string here

File: src/renderer/src/atoms/sidebar.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { createAtomHooks } from "@renderer/lib/jotai"
+import { atom } from "jotai"
+
+export const [
+  ,
```
Comment (zhangmo8) [inline src/renderer/src/atoms/sidebar.ts:5]: remove this

File: src/renderer/src/components/ui/markdown/plugins/container.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  getMd: () => MarkdownIt,
+): ContainerArgs {
+  return [
+    container,
+    klass,
+    {
+      render(tokens, idx, _options, env: { references?: any }) {
+        const md = getMd()
+        const token = tokens[idx]
+        const info = token.info.trim().slice(klass.length).trim()
```
Comment (zhangmo8) [inline src/renderer/src/components/ui/markdown/plugins/container.ts:19]:
```suggestion
        const info = token.info.trim()?.slice(klass.length)?.trim()
```

Comment (Innei) [reply src/renderer/src/components/ui/markdown/plugins/container.ts:19]: This code was copied from vitepress, there are no problems here.

**Review by zhangmo8 — APPROVED** [review]
LGTM

**Review by DIYgod — APPROVED** [review]

**Pull request merged**